src/ansys/fluent/core/physicalquantities/__init__.py

```diff
@@ -0,0 +1,53 @@
+# Copyright (C) 2021 - 2025 ANSYS, Inc. and/or its affiliates.
```

Comment:
Search engine optimization techniques suggest that we should avoid a long name to a file. We can use `physical_quantities.py` here as it also helps to improve both search performance and the readability. We had done the similar change to `system_coupling.py`.